src/main/resources/entity-types/openaccess/drv_openaccess_party.json5

```
@@ -0,0 +1,57 @@
+{
+  id: '0f8f6d6f-504f-4aa9-9d98-9779aaaa907b',
+  name: 'drv_openaccess_party',
+  private: false,
+  fromClause: 'src_oa_party AS party\nLEFT JOIN src_oa_alternate_email_address ON alternate_email_address.aea_owner_fk=party.p_id\nLEFT JOIN src_oa_refdata_value AS p_institution_level_1 ON p_institution_level_1.rdv_id=party.p_institution_level_1_fk',
```

Since this is JSON5, you can add line breaks to make this more readable
```suggestion
  fromClause: 'src_oa_party AS party \
    LEFT JOIN src_oa_alternate_email_address ON alternate_email_address.aea_owner_fk=party.p_id \
    LEFT JOIN src_oa_refdata_value AS p_institution_level_1 ON p_institution_level_1.rdv_id=party.p_institution_level_1_fk',
```

---

I entered line break in the entity type creator, and it changed it to "\n" automatically.. 

when I tried changing this to line breaks in the source file, it didn't work (I used line breaks plus indentation, maybe that doesn't work?)

but for me no big deal here..